fairroot/eventdisplay/FairEventManager.h

```diff
@@ -151,7 +156,6 @@ class FairEventManager : public TEveEventManager
     TEveProjectionAxes* GetRPhiAxes() const { return fAxesPhi; };
     TEveProjectionAxes* GetRhoZAxes() const { return fAxesRho; };
     virtual void LoadXMLSettings();
-    void LoadXMLDetector(TGeoNode* node, FairXMLNode* xml, Int_t depth = 0);
```

Comment:
This is a public member function. So you're removing public API.

This is a breaking change.

If you really want this breaking change, then please write an appropriate CHANGELOG entry.

Reply:
I'm not sure if I made change corretly, please check :)